Potentiometer/CS/README.md

```diff
@@ -0,0 +1,305 @@
+- ADC
+    - Upboard
+        - [1 MCP3008 10-bit ADC](http://www.microchip.com/wwwproducts/Devices.aspx?dDocName=en010530) or [1 MCP3208 12-bit ADC](http://www.digikey.com/product-search/en?KeyWords=mcp3208%20ci%2Fp&WT.z_header=search_go)
+        - [1 Voltage-Level Translator Breakout](https://www.sparkfun.com/products/11771)
+- [1 10k &#x2126; Trimmer Potentiometer](http://www.digikey.com/product-detail/en/3362P-1-103TLF/3362P-103TLF-ND/1232540)
+- Upboard
+- 1 breadboard and a couple of wires
+- HDMI Monitor and HDMI cable
+
+## Parts Review
+
+In this sample, you have the option of using either the MCP3002 or MCP3208 ADC (Analog to Digital Converter). 
+The differences between the chips are the number of input channels and resolution. 12-bit resolution is more accurate than the 10-bit options, and the number of channels determines how many different inputs you can read.
+
+Below are the pinouts of the MCP3002 and MCP3208 ADCs. 
+
+| MCP3002                                                              | MCP3008 or MCP3208                                                              |
+| -------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| ![MCP3002 Pinout](../../Resources/MCP3002.png) | ![MCP3208 Pinout](../../Resources/MCP3208.png) |
+
+
+#### Upboard Pinout
+
+![UpBoard pinout](../../Resources/Upboard_Pinout.png)
+
+#### Wiring & Connections
+
+##### MCP3002
+If you chose to use the **MCP3002**, assemble the circuit as follows. Note that the wiper pin (the middle pin on the 10k potentiometer) should be connected to `CH0` on MCP3002. You can also refer to the [datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/21294E.pdf) for more information.
+
+Detailed connection:
+
+![Overall Schematics](../../Resources/OverallCon-3002.png)
+
+The MCP3002 should be connected as follows:
+
+- MCP3002: VDD/VREF - 3.3V on UPBOARD
+- MCP3002: CLK - "SPI0 SCLK" on UPBOARD
+- MCP3002: Dout - "SPI MISO" on UPBOARD
+- MCP3002: Din - "SPI MOSI" on UPBOARD
+- MCP3002: CS/SHDN - "SPI CS0" on UPBOARD
+- MCP3002: Vss - GND on UPBOARD
+- MCP3002: CH0 - Potentiometer wiper pin
+
+
+##### MCP3208 or MCP3008
+If you chose to use the **MCP3208** or **MCP3008**, assemble the circuit as follows. Note that the wiper pin (the middle pin on the 10k potentiometer) should be connected to `CH0` on MCP3208. You can also refer to the [MCP3208 datasheet](http://pdf.datasheetcatalog.com/datasheets2/43/435228_1.pdf) or the [MCP3008 datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/21295C.pdf) for more information.
+
+Detailed connection:
+
+![Overall Schematics](../../Resources/OverallCon-3208.png)
+
+The MCP3208 should be connected as follows:
+
+- MCP3208: VDD - 3.3V on UPBOARD
+- MCP3208: VREF - 3.3V on UPBAORD
+- MCP3208: AGND - GND on UPBAORD
+- MCP3208: CLK - "SPI SCLK" on UPBAORD

```

Comment:
UPBOARD is miss-spelt UPBAORD here and a few other places.  Also, should it be "UpBoard"?

Reply:
Correct the Spelling to UPBOARD in readme document.